WORKSPACE

```diff
@@ -12,7 +12,8 @@ load("@bazel_skylib//lib:versions.bzl", "versions")
 
 versions.check("3.4.1")
 
-rules_ruby_select_sdk("3.0.2")
+load("@rules_ruby//ruby:deps.bzl", "rules_ruby_register_toolchains")
+rules_ruby_register_toolchains(["ruby-3.0", "jruby-9.2"])
```

Comment:
Am I reading this correctly - this sets defaults, but doesn't restrict the list of supported versions?

Reply:
No, this restricts the available versions of ruby in the current WORKSPACE (it will download and build them all).  This can be modified as needed though, and doesn't affect the versions supported by rules_ruby.